CoreWebServer/src/au/gov/asd/tac/constellation/webserver/resources/constellation_client.py

```diff
@@ -20,7 +20,7 @@
 # For example, if a new function is added, clients that require that function
 # to be present can check the version.
 #
-__version__ = 20200326
+__version__ = 20200327
```

Comment:
I think this was a datetime stamp :)

Reply:
` Increase this version number when a change happens.` This comment made me think I should manually increase it. Will remove.

Reply:
I think its a manual change. I was mainly noting that the version number is actually a datetime stamp probably representing when it was last changed (i.e 2020-03-26). So suggesting the version number should probably be changed to something more like 20231115 for instance